docs: Clarify it's an installer on Windows instructions (#14172)
kubernetes/minikube: site/content/en/docs/start/_index.md

{{% /quiz_instruction %}}

{{% quiz_instruction id="/Windows/x86-64/Stable/.exe download" %}}

> 1. Download the installer of the [latest release](https://storage.googleapis.com/minikube/releases/latest/minikube-installer.exe) and run it.  

Reviewer: ```suggestion
1. Download and run the installer for the [latest release](https://storage.googleapis.com/minikube/releases/latest/minikube-installer.exe).
```

<br>
    Or if using `PowerShell`, use this command:
    ```powershell
    New-Item -Path 'c:\' -Name 'minikube' -ItemType Directory -Force
    Invoke-WebRequest -OutFile 'c:\minikube\minikube.exe' -Uri 'https://github.com/kubernetes/minikube/releases/latest/download/minikube-windows-amd64.exe' -UseBasicParsing
    ```

> 2. Add the `minikube.exe` binary in to your `PATH`.  

Reviewer: ```suggestion
2. Add the `minikube.exe` binary to your `PATH`.
```

{{% /quiz_instruction %}}

{{% quiz_instruction id="/Windows/x86-64/Beta/.exe download" %}}

> 1. Download the installer of the <a href="#" id="latest-beta-download-link">latest beta release</a> and run it.  

Reviewer: ```suggestion
1. Download and run the installer for the <a href="#" id="latest-beta-download-link">latest beta release</a>.
```

    Invoke-WebRequest -Uri $item.browser_download_url -OutFile 'c:\minikube\minikube.exe' -UseBasicParsing
    ```

> 2. Add the `minikube.exe` binary in to your `PATH`.  

Reviewer: ```suggestion
2. Add the `minikube.exe` binary to your `PATH`.
```
